dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    xmlns:local1="using:MUXControlsTestApp.Scripts">
```

Comment:
>local1 [](start = 10, length = 6)

localConverters would be better.

Reply:
or Maybe PagerConverters, I'm not sure.

---
In reply to: [463300145](https://github.com/microsoft/microsoft-ui-xaml/pull/2977#discussion_r463300145) [](ancestors = 463300145)

Reply:
Agreed, I have changed the names